tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -1,99 +1,57 @@
 import { myUser, opponentOne } from '../../fixtures/userFixtures';
 
-function setup(isRanked = false) {
-  cy.wipeDatabase();
-  cy.visit('/');
-  cy.signupPlayer(myUser);
-  cy.createGamePlayer({ gameName: 'Test Game', isRanked }).then((gameSummary) => {
-    cy.window()
-      .its('cuttle.gameStore')
-      .then((store) => store.requestSubscribe(gameSummary.gameId));
-    cy.vueRoute(`/lobby/${gameSummary.gameId}`);
-    cy.wrap(gameSummary).as('gameSummary');
-  });
-}
 
-function checkRanked(ranked) {
-  cy.window()
-  .its('cuttle.gameStore')
-    .then((game) => {
-      expect(game.isRanked).to.eq(ranked);
-    });
-}
+import en from '../../../../src/translations/en.json';
```

Comment:
I think it makes sense to just use the words instead of the translations from the file, considering most of them are just one word and we are only pulling english

Reply:
OK.  My idea here was that if we modify/correct the displayed text, we'll need to update all the tests too.  Are we testing the actual text ? or just the fact that the text gets displayed and that it's fetched from the correct source ?

Reply:
I agree that if we made changes we'd have to come back to the test and edit it, but I don't think we use the translations in any other test files except the specific localization files and for the sake of consistency we should avoid it here. I think using the translations could possibly be something we can look into, but on a separate PR